src/components/Outline.tsx

```diff
@@ -39,8 +47,8 @@ export function Outline() {
           <p className="text-gray-500 dark:text-gray-400">Current Net Worth</p>
         </div>
```

Comment:
This collection of cards should definitely follow the pattern you used in the expenses. Ideally each file is as reusable and clean as possible. This is especially true when it comes to the JSX portion. The more concise it is, the better.